index.bs

```diff
@@ -2495,7 +2495,7 @@ partial dictionary MLOpSupportLimits {
 </dl>
 
 <div class="note">
-    A *depthwise* conv2d operation is a variant of grouped convolution, used in models like the MobileNet, where the *options.groups* = inputChannels = outputChannels and the shape of filter tensor is *[options.groups, 1, height, width]*
+    A *depthwise* conv2d operation is a variant of grouped convolution, used in models like the MobileNet, where the *options*.{{MLConv2dOptions/groups}} = inputChannels = outputChannels and the shape of filter tensor is *[options.groups, 1, height, width]*
```

Comment:
Should `inputChannels` and `outputChannels` be formatted? In `*[options.groups, 1, height, width]*`, should we use `[*options*.{{MLConv2dOptions/groups}}, 1, *height*, *width*]`?

I found the prose of `MLConv2dOptions.inputLayout` and `MLConv2dOptions.filterLayout` also refer to variables including `inputChannels` and `outputChannels` etc.,. Should we also format them separately?

Reply:
Formatted `*inputChannels*` and `*outputChannels*` in that instance (in f714fc1a5c5a7b81ef1370341df296cec562c70f) . As noted above, need to decide more generally on how to format shapes.